src/AspNetCoreModuleV2/InProcessRequestHandler/managedexports.cxx

```diff
@@ -169,16 +177,15 @@ http_get_application_properties(
     _In_ IISConfigurationData* pIISCofigurationData
 )
 {
-    REQUESTHANDLER_CONFIG* pConfiguration = NULL;
-    IN_PROCESS_APPLICATION* pApplication = IN_PROCESS_APPLICATION::GetInstance();
-
-    if (pApplication == NULL)
+    auto pInProcessApplication = IN_PROCESS_APPLICATION::GetInstance();
```

Comment:
Are we making it a rule that we don't throw with pInvokes from managed into native?

Reply:
Yeah, let's not leak exception in any direction of managed<->native boundary.